tests/test_mfa.py

```diff
@@ -169,34 +180,34 @@ def test_challenge_success(
         )
         assert challenge_factor == mock_challenge_factor_response
 
-    def test_verify_factor_no_id(self, mock_verify_challenge_payload):
+    def test_verify_challenge_no_id(self, mock_verify_challenge_payload):
         with pytest.raises(ValueError) as err:
-            self.mfa.verify_factor(
+            self.mfa.verify_challenge(
```

Comment:
We want to keep the `verify_factor` tests in place to ensure the `verify_factor` method is still under test.

We already have test coverage for the `verify_challenge` method.

We can remove these tests when we remove the `verify_factor` method in the next major version.

Reply:
That makes sense! I just removed those changes. ☑️